Document custom gateway usage (#163)
kyma-project/docker-registry: docs/user/technical-reference/resources/06-20-docker-registry-cr.md

**Spec:**

| Parameter                               | Type   | Description                                                                                                                |
|-----------------------------------------|--------|----------------------------------------------------------------------------------------------------------------------------|
| **externalAccess**                      | object | Contains configuration of the registry external access through the Istio Gateway.                                          |
| **externalAccess.enabled**              | string | Specifies if registry is exposed.                                                                                          |

> | **externalAccess.gateway**              | string | Specifies the name of the Istio Gateway CR in format NAMESPACE/NAME. Defaults to the `kyma-system/kyma-gateway`.           |

Reviewer: ```suggestion
| **externalAccess.gateway**              | string | Specifies the name of the Istio Gateway CR in the `NAMESPACE/NAME` format. Defaults to the `kyma-system/kyma-gateway`.           |
```

> | **externalAccess.host**                 | string | Specifies the host on which the registry will be exposed. It needs to fit into at least one server defined in the Gateway. |

Reviewer: ```suggestion
| **externalAccess.host**                 | string | Specifies the host on which the registry will be exposed. It must fit into at least one server defined in the Gateway. |
```
